apps/server/src/modules/common-cartridge/service/common-cartridge-export.service.ts

```diff
@@ -31,4 +33,16 @@ export class CommonCartridgeExportService {
 
 		return courseRooms;
 	}
+
+	public async getLessonById(lessonId: string) {

```

Comment:
Return type is missing.

Reply:
this should be deleted after review

Reply:
Ok, I will resolve this conservation when changes are reverted.